### PR TITLE
Citizen Development blog article

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Citizen Development blog article

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2023/10/citizen-development.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Innovate from within - Why manufacturing must embrace Citizen Developers
+subtitle: Empower your Operational Technology teams as Citizen Developers
+authors: ["marian-demme"]
+#image: blog/2023/10/images/tile-blog-custom-vuetify-components-for-Dashboard.png
+date: 2023-10-23
```
Comment (ZJvandeWeg) [inline src/blog/2023/10/citizen-development.md:6]: @gdziuba @MarianRaphael I think "Build vs Buy" for MES systems should go out tomorrow. Would you be OK with publishing say Wednesday @MarianRaphael ?

File: src/blog/2023/10/citizen-development.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+This shift toward embracing citizen development within manufacturing represents a unique opportunity where both sides benefit significantly; not only do OT departments gain access to custom applications tailored precisely for their needs, but IT teams can lead guidance for self-empowerment, resulting in greater efficiency through automation.
+
+## Democratize Application Development with Node-RED
+
+For the past decade, Node-RED has been a pioneer in low-code tools, democratizing application development in manufacturing. It provides powerful data collection, transformation, and visualization capabilities, simplifying the application-building process. As an open-source tool with no associated costs or licensing fees, Node-RED is [increasingly popular](blog/2023/03/integration-platform-for-edge-computing/#the-standard-for-edge-computing-and-plcs) among manufacturers seeking to quickly develop custom applications without extensive coding knowledge or experience. FlowFuse takes things one step further, offering organizations a way to officially incorporate and scale Node-RED, making it compliant, governed, and secure within the existing solution landscape.
```
Comment (ZJvandeWeg) [inline src/blog/2023/10/citizen-development.md:37]:
```suggestion
For the past decade, Node-RED has been a pioneer in low-code tools, democratizing application development in manufacturing. It provides powerful data collection, transformation, and visualization capabilities, simplifying the application-building process.
As an open-source tool with no associated costs or licensing fees, Node-RED is [increasingly popular](/blog/2023/03/integration-platform-for-edge-computing/#the-standard-for-edge-computing-and-plcs) among manufacturers seeking to quickly develop custom applications without extensive coding knowledge or experience.
FlowFuse takes things one step further, offering organizations a way to officially incorporate and scale Node-RED, making it compliant, governed, and secure within the existing solution landscape.
```

Comment (ZJvandeWeg) [reply src/blog/2023/10/citizen-development.md:37]: The slash on the URL is my main point

File: src/blog/2023/10/citizen-development.md (hunk 1 above)
Comment (MarianRaphael) [reply src/blog/2023/10/citizen-development.md:6]: Sure, it's also okay to wait until next week to have content during our summit.

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2023/10/citizen-development.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+title: Innovate from within - Why manufacturing must embrace Citizen Developers
+subtitle: Empower your Operational Technology teams as Citizen Developers
+authors: ["marian-demme"]
+image: blog/2023/10/images/image-citizen-developers.png
+date: 2023-10-23
```
Comment (ZJvandeWeg) [inline src/blog/2023/10/citizen-development.md:6]:
```suggestion
date: 2023-10-30
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**